bot/api/client/src/main/kotlin/ClientDefinitionBuilders.kt

```diff
@@ -60,20 +57,20 @@ fun newBot(
     )
 
 /**
- * Creates a new story.
+ * Creates a new story [ClientStoryDefinition] in bot Api mode
+ * @param mainIntent [String] The main intent name.
```

Comment:
Is specifying the type really necessary here?

Reply:
Not sure but there is the one with IntentAware below.